platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -38,65 +39,65 @@ public BaseUtilitiesTest(String name) {
     @Override
     protected void setUp() throws Exception {
         super.setUp();
-        BaseUtilities.resetOperatingSystem ();
+        BaseUtilities.resetOperatingSystem();
         originalOsName = System.getProperty("os.name");
     }
-    
+
     @Override
     protected void tearDown() throws Exception {
         System.setProperty("os.name", originalOsName);
         super.tearDown();
     }
 
-    public void testGetOperatingSystemWinNT () {
-        System.setProperty ("os.name", "Windows NT");
+    public void testGetOperatingSystemWinNT() {
+        System.setProperty("os.name", "Windows NT");
         //assertEquals ("System.getProperty (os.name) returns Windows NT", "Windows NT", System.getProperty ("os.name"));
-        assertEquals ("Windows NT recognized as OS_WINNT", BaseUtilities.OS_WINNT, BaseUtilities.getOperatingSystem ());
+        assertEquals("Windows NT recognized as OS_WINNT", BaseUtilities.OS_WINNT, BaseUtilities.getOperatingSystem());
```

Comment:
Reformatting existing code makes reviews harder. We don't want to reformat existing code without a real need. Otherwise we'll be revieweing reformats for each one of the NetBeans committers.

Reply:
Proper formating is important but I get the idea. Next Time if I reallt, really need to format something I will provide PR just with formating change.

Reply:
there is a little trick you can do if you really really want to sneak reformats into a PR which is not related: you keep them in a separate commit and mention it in the PR text (but don't overdo that). So reviewers can concentrate on the non-stylistic changes, but still glance over the formatting commit and look at it separately from the actual issue. Reducing noise is always helpful.